src/VirtoCommerce.Platform.Security/Repositories/SecurityDbContext.cs

```diff
@@ -52,7 +52,7 @@ public override int SaveChanges(bool acceptAllChangesOnSuccess)
         }
         public override Task<int> SaveChangesAsync(CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.SaveChangesWithTriggersAsync(base.SaveChangesAsync, acceptAllChangesOnSuccess: true, cancellationToken: cancellationToken);
```

Comment:
I asked to remove only `cancellationToken:` because `cancellationToken: cancellationToken` is redundant.
But you shouldn't remove `acceptAllChangesOnSuccess:`.